espefuse.py

```diff
@@ -30,186 +30,5 @@
-    init_parser.add_argument('--path-efuse-file', help='For host tests, saves efuse memory to file.', type=str, default=None)
-    init_parser.add_argument('--do-not-confirm', help='Do not pause for confirmation before permanently writing efuses. Use with caution.', action='store_true')
-
-    common_args, remaining_args = init_parser.parse_known_args(custom_commandline)
-    debug_mode = common_args.debug or ("dump" in remaining_args)
-    just_print_help = [True for arg in remaining_args if arg in ["--help", "-h"]] or remaining_args == []
-
-    print("espefuse.py v{}".format(esptool.__version__))
-
-    try:
-        esp = get_esp(
-            common_args.port,
-            common_args.baud,
-            common_args.before,
-            common_args.chip,
-            just_print_help,
-            common_args.virt,
-            common_args.debug,
-            common_args.path_efuse_file,
-        )
-    except esptool.FatalError as e:
-        raise esptool.FatalError(
-            "{}\nPlease make sure that you have specified "
-            "the right port with the --port argument".format(e)
-        )
-    efuses, efuse_operations = get_efuses(esp, just_print_help, debug_mode, common_args.do_not_confirm)
-
-    parser = argparse.ArgumentParser(parents=[init_parser])
-    subparsers = parser.add_subparsers(dest='operation', help='Run espefuse.py {command} -h for additional help')
-
-    efuse_operations.add_commands(subparsers, efuses)
-
-    grouped_remaining_args, used_cmds = split_on_groups(remaining_args)
-
-    if len(grouped_remaining_args) == 0:
-        parser.print_help()
-        parser.exit(1)
-    there_are_multiple_burn_commands_in_args = sum(cmd in SUPPORTED_BURN_COMMANDS for cmd in used_cmds) > 1
-    if there_are_multiple_burn_commands_in_args:
-        efuses.batch_mode_cnt += 1
-
-    for rem_args in grouped_remaining_args:
-        args, unused_args = parser.parse_known_args(rem_args, namespace=common_args)
-        if args.operation is None:
-            parser.print_help()
-            parser.exit(1)
-        assert len(unused_args) == 0, 'Not all commands were recognized "{}"'.format(unused_args)
-
-        operation_func = vars(efuse_operations)[args.operation]
-        # each 'operation' is a module-level function of the same name
-        print('\n=== Run "{}" command ==='.format(args.operation))
-        operation_func(esp, efuses, args)
-
-    if there_are_multiple_burn_commands_in_args:
-        efuses.batch_mode_cnt -= 1
-        if not efuses.burn_all(check_batch_mode=True):
-            raise esptool.FatalError('BURN was not done')
-
-    if common_args.virt is False:
-        esp._port.close()
-
-
-def _main():
-    try:
-        main()
-    except esptool.FatalError as e:
-        print('\nA fatal error occurred: %s' % e)
-        sys.exit(2)
-
-
-if __name__ == '__main__':
-    _main()
+if __name__ == "__main__":
+    espefuse._main()
```

Comment:
most of the file was removed since it was a duplicate of what was in the `espefuse/__init__.py`